app/models/concerns/encrypt_field.rb

```diff
@@ -0,0 +1,75 @@
+module EncryptField
```

Comment:
`Encryptable` module encrypts the whole attribute (column), while in some cases there is a  need to encrypt a specific  field (instance varialbe). such use cases can be found in settings, and in parameters. 
hence `encryptable` module is splitted into two modules, the first one is `encrypt_field`, which can be included to models in case of specific fields encryption, and the `encryptable` which keeps the logic for attribute encryption.